examples/db-templates/mongodb-ephemeral-template.json

```diff
@@ -49,7 +49,11 @@
       "spec": {
         "strategy": {
           "type": "Recreate",
-          "resources": {}
+            "resources": {
+		"limits": {
+		    "memory": "${MEMORY_LIMIT}"
```

Comment:
Yeah, I've carried this since early trials, as at that time it needed be set for the deployment to even start.  I think used the one value to keep things simple for the user, at least wrt this stuff.

I'll revisit the need for it and update the PR with what I find.